src/code-templates/services/order-service/entry-points/api/routes.ts

```diff
@@ -1,15 +1,21 @@
 import util from "util";
 import express from "express";
 import * as newOrderUseCase from "../../domain/new-order-use-case";
+import { AsyncLocalStorage } from 'async_hooks';
 import { errorHandler } from "@practica/error-handling";
 
-export const defineRoutes = (expressApp: express.Application) => {
+export const defineRoutes = (expressApp: express.Application, asyncLocalStorage: AsyncLocalStorage<Map<string, string>>) => {
```

Comment:
Since the same functionality, getting the store, is needed in many other modules (e.g. DB access file), we can't pass als (asynclocalstorage) to every function. Consequently, we need to hold a singleton instance in a module so every potential caller can use it

e.g., module.exports = new AsyncLocalStorage

Then every caller who require it will get the singleton